ecosystem/indexer-grpc/indexer-grpc-utils/src/config.rs

```diff
@@ -20,30 +20,59 @@ pub struct IndexerGrpcConfig {
     pub health_check_port: u16,
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize, Default)]
+#[serde(deny_unknown_fields)]
+pub struct IndexerGrpcProcessorConfig {
```

Comment:
Add some comments on what this is.

Reply:
And also some comments on IndexerGrpcConfig.

Reply:
done.